hadoop-hdds/common/src/main/java/org/apache/hadoop/utils/db/RDBTable.java

```diff
@@ -183,4 +183,14 @@ public String getName() throws IOException {
   public void close() throws Exception {
     // Nothing do for a Column Family.
   }
+
+  @Override
+  public long getEstimatedKeyCount() throws IOException {
+    try {
+      return db.getLongProperty(handle, "rocksdb.estimate-num-keys");
+    } catch (RocksDBException e) {
+      throw new IOException(
+          "Failed to get estimated key count of the table.");
```

Comment:
Minor Nit: Can we add the table name here.

Reply:
Sure! Done.